### ISSUE TITLE
Timeline cannot be reached by keyboard users

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-web#10902]
> No keyboard access to the message list.
> o	Screen reader users can navigate and interact with the message list using browse mode. However, keyboard-only users who don't use a screen reader won't be able to do this.
> o	There's no indication that an autocomplete appeared.
> o	At least they appear near the text box.
> o	Screen reader users would need to interact with them using browse mode.

**Comment by t3chguy** [comment]
> https://github.com/vector-im/riot-web/issues/10031

is fixed by https://github.com/matrix-org/matrix-react-sdk/pull/3497

**Comment by t3chguy** [comment]
Blocked on product decisions in https://github.com/vector-im/riot-web/issues/11075

**Comment by chagai95** [comment]
Is this still blocked?

**Comment by t3chguy** [comment]
Yup, there are fundamental questions about how product wants to make keyboard navigation of the app work relating to accessibility such as https://github.com/vector-im/element-web/issues/11071

**Comment by dbkr** [comment]
We finally closed #11071 so I think this can be closed now
